lib/web_ui/test/window_test.dart

```diff
@@ -70,6 +70,7 @@ void testMain() {
       'flutter/navigation',
       JSONMethodCodec().encodeMethodCall(MethodCall(
         'routeUpdated',
+        // ignore: prefer_const_literals_to_create_immutables
```

Comment:
same question here. Why not just use const literals here?

Reply:
Fixed.